05 lection3/02 modal

lk4n opened the pull request:
Привет.
JS для body не работал, исправил: 
`    document.body.overflow = 'hidden'; ` на `
    document.body.style.overflow = 'hidden';`
Добавил базовую aria для кнопки `modal__close` и body. Как я понял, арию `aria-hidden="true"` для элементов с `display:none` не надо добавлять, они все равно не отрисовуются, по этому для блока modal его не ставил.

lk4n committed:
Fix font typos

lk4n committed:
Initial

lk4n committed:
Update step 1

lk4n committed:
Update padding and box-shadow

lk4n committed:
Update step 2

Trying to sort components by type
Font mixin must contain font-weight
Shorter containers' block margin syntax

lk4n committed:
Move to sass

lk4n committed:
Move to sass

lk4n committed:
Merge branch '04-lection2/02-skeleton' into 05-lection3/02-modal

lk4n committed:
Basic styling

lk4n committed:
Fix javascript for body

lk4n committed:
Update

added to html basic aria
fully style dialog

lk4n committed:
Change in html structure

Add new element for modal dialog. Can't make dialog and overlay on single element
Remove overlay color using transparent black instead

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

GSemikozov commented on components/modal/_modal.scss:
```diff
@@ -0,0 +1,50 @@
+.modal {
+  display: none;
+
+  &:before {
```
не через before - сама модалка должна позиционироваться как fixed )

lk4n replied:
Исправил.

lk4n committed:
Move overlay from pseudo element

lk4n committed:
Update html and js

There are 2 sections: .main and .modal. Section .main aria-hidden if .modal shown.
Initial focus set to .modal__title

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.
